src/styles.scss

```diff
@@ -139,11 +157,12 @@ mat-grid-tile.tableTitle {
 .contentButton {
```

Comment:
This would change these buttons globally, across the entire app. Perhaps it'd be better to create a new CSS class for this if you want a white button with bigger font?